streams/cluster/src/test/java/com/hortonworks/streamline/streams/cluster/register/impl/HBaseServiceRegistrarTest.java

```diff
@@ -53,6 +56,11 @@ public void testRegister() throws Exception {
             ManualServiceRegistrar.ConfigFileInfo hiveSiteXml = new ManualServiceRegistrar.ConfigFileInfo(HBASE_SITE_XML, is);
             registrar.register(cluster, new Config(), Lists.newArrayList(hiveSiteXml));
         }
+
+        Service hbaseService = environmentService.getServiceByName(cluster.getId(), HBaseServiceRegistrar.SERVICE_NAME_HBASE);
+        assertNotNull(hbaseService);
```

Comment:
it's beyond the scope of this patch, but I would really vote that we get rid of all of these nulls. Do you think we should file a JIRA for that ?